client/state/themes/reducer.js

```diff
@@ -117,7 +169,7 @@ export const queries = ( () => {
 			[ siteId ]: nextManager
 		};
 	}
-
+	/* eslint-disable no-shadow */
```

Comment:
Let's not disable this warning here. The shadowing will only be temporary anyway; it will go away once we remove the old `themes` reducer.